Pdf download (#18)
Xceptance/neodymium-showcase: README.md

In `CheckboxTest.java` there is are more sophisticated usecases for shadow DOM including nested shadow DOM's.
## 2. Localization
Under `showcase.neodymium.tests.localization` you can find how to use the localization feature of neodymium. Note that all the translated strings can be found in the `localization.yaml` in the config folder.
## 3. PDF Download

> In the `showcase.neodymium.tests.pdf` package is a basic test for downloading and checking a PDF file. In this test we use the pdfbox framework for handeling the PDF file.
> The selenide framework contains the `download` function. With it you can download a file directly via a given link.
> Then you have to load the PDF file into a document that can be handled by pdfbox. This is the `PDDocument` class. With this class you can extract any information from the PDF.
> In the example we simply extract all the text of the PDF, with the `PDFTextStripper` class, and compare it with an expected value.

Reviewer: ```suggestion
In the `showcase.neodymium.tests.pdf` package is a basic test for downloading and checking a PDF file. In this test, we use the PDFBox framework for handling a PDF file.
The Selenide framework contains the `download` function which allows to download a file directly via a given link.
Then, you have to load the PDF file into a document that can be handled by PDFBox. This is the `PDDocument` class. With this class, you can extract any information from the PDF.
In the example, we extract the whole text of the PDF with the `PDFTextStripper` class and compare it with an expected value.
```
1. Stick with the correct spelling for a framework.
1. Grammar fixes
1. Spelling fixes

However, I'm unsure whether or not we really want to supply a step by step instruction on how to use PDFBox. Maybe we should just provide a high level description, e.g. something along the line of "download a file, open it with PDFBox and get the text which we can then verify".

Author: I edited the README to be consitent with the framework names. Also i replaced the instruction on how to use PDFBox with a more highlevel descritpion.

Reviewer: I'm still not content with your description as the "you" in our README refers to the reader. This makes the following a bit awkward:
> Then, you have to load the PDF file with the PDFBox framework. With it we can get to the text and verify it.

Since the reader isn't the one to actually load the PDF file, I'd rather go with something along the line of:
`Then, the PDF file is loaded with the PDFBox framework, so we can access the text and verify it.`

What do you think?

Author: I don't have a strong opinion on this. I edited to be your suggestion.

## 2. Localization
Under `showcase.neodymium.tests.localization` you can find how to use the localization feature of neodymium. Note that all the translated strings can be found in the `localization.yaml` in the config folder.
## 3. PDF Download
In the `showcase.neodymium.tests.pdf` package is a basic test for downloading and checking a PDF file. In this test, we use the PDFBox framework for handling a PDF file.

> The Selenide framework contains the `download` function which allows to download a file directly via a given link.

Reviewer: This sounds a bit out of place since there is no association with the previous sentence. Hence, I suggest the following
```suggestion
To first download such a file, we use Selenide's `download` function in order to download a file directly via a given link.
```
